meta-resin-common/recipes-containers/mkfs-hostapp-native/files/mkfs.hostapp-ext4

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o errexit
+set -x
```

Comment:
guess this was used for development and can be dropped now?

Reply:
yep. done